web-app/src/components/tournament/creator/tournamentTypeSelector.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div class="row col-12 justify-center">
+    <selectable-tournament-type
+      v-for="item in tournamentTypesOptions"
+      :key="item.type"
+      :tournamentTypeName="item.label"
+      :tournamentType="item.type"
```

Comment:
try to pass only `tournamentType` and store locales as e.q. `tournament.type.single-elimination`. You will be able to get locales with `$t('tournament.type.' + tournamentType)`

in locales files:
"single-elimination": "txt"